src/features/account/TokenListItem.tsx

```diff
@@ -119,31 +157,171 @@ const TokenListItem = ({ mint }: Props) => {
   )
 }
 
-export default TokenListItem
+export const TokenListGovItem = ({ mint }: { mint: PublicKey }) => {
+  const navigation = useNavigation()
+  const { anchorProvider, connection } = useSolana()
+  const wallet = useCurrentWallet()
+  const { triggerImpact } = useHaptic()
+  const { json, symbol, loading } = useMetaplexMetadata(mint)
+  const decimals = useMint(mint)?.info?.decimals
+  const mintStr = mint.toBase58()
+  const colors = useColors()
+
+  const args = useMemo(
+    () =>
+      wallet &&
+      mint &&
+      connection && {
+        wallet: anchorProvider?.wallet.publicKey,
+        mint,
+        provider: anchorProvider,
+      },
+    [mint, wallet, connection, anchorProvider],
+  )
+
+  const { result, loading: loadingPositionKeys } = useAsync(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    async (args: any | undefined) => {
+      if (args) {
+        return getPositionKeys(args)
```

Comment:
This is going to be really inefficient, because it's getting called once per token list item. In the background it's fetching all the metaplex NFTs and filtering down to the ones with the right `registrar.collection`.  Can you see if Helius' new token API is capable of getting these position NFTs by collection? I think it is. If so can make this a lot faster. Just update the `getPositionKeys` impl